lib/heex_formatter/phases/tokenizer.ex

```diff
@@ -41,9 +41,12 @@ defmodule HeexFormatter.Phases.Tokenizer do
   end
 
   defp tokenize({:text, _line, _column, text}, acc) do
-    string = List.to_string(text)
-    {tokens, :text} = HTMLTokenizer.tokenize(string, "nofile", 0, [], [], :text)
-    {tokens |> Enum.reject(&line_break?/1) |> Enum.reverse(), acc}
+    string = IO.iodata_to_binary(text)
+
+    case HTMLTokenizer.tokenize(string, "nofile", 0, [], [], :text) do
+      {tokens, :text} -> {tokens |> Enum.reject(&line_break?/1) |> Enum.reverse(), acc}
```

Comment:
maybe extract it to a function as we're using in the line below as well? like:

```elixir
remove_line_breaks_on_tokens = &(&1 |> Enum.reject(&line_break?/1) |> Enum.reverse())

remove_line_breaks_on_tokens.(tokens)

# or

defp remove_line_breaks_on_tokens(tokens) do
  tokens |> Enum.reject(&line_break?/1) |> Enum.reverse()
end
```

Reply:
Is reverse part of "remove line breaks"?

Reply:
What do you think about the new version?

Reply:
good point, this seems to be specific to that part of the code, so maybe a capture function? This way we need to update only one place if we need to change the logic 😅

Reply:
At the end the case changed to a single case that matches `{tokens, _}`.

Reply:
Looks great 😊

Reply:
TY